.github/workflows/release-airbyte-os.yml

```diff
@@ -77,6 +84,27 @@ jobs:
           CLOUDREPO_PASSWORD: ${{ secrets.CLOUDREPO_PASSWORD }}
         run: |
           ./tools/bin/release_version.sh
+      
+      - name: Commit and push changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: 'Bump subchart version to ${{ steps.release_airbyte.outputs.NEW_VERSION }}'
+          add: '.'
+          cwd: './github/helm-charts'
+
+      - name: Release main helm chart
+        shell: bash
+        run: |
+          cd ./charts/airbyte && helm dep update && cd -
+          helm package ./charts/airbyte -d ./.github/helm-charts --version ${{ steps.release_airbyte.outputs.NEW_VERSION }}
+          helm repo index ./.github/helm-charts/
+      
+      - name: Commit and push changes
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: 'Bump main chart version to ${{ steps.release_airbyte.outputs.NEW_VERSION }}'
+          add: '.'
+          cwd: './github/helm-charts'
```

Comment:
should all of this happen in an action? 

in that way we can reuse this in the publish for Cloud workflow